extension/src/uddsketch.rs

```diff
@@ -582,6 +581,46 @@ pub fn uddsketch_approx_percentile<'a>(percentile: f64, sketch: UddSketch<'a>) -
     )
 }
 
+// #[pg_operator(immutable, schema = "toolkit_experimental", parallel_safe)]
```

Comment:
Note from standup : You cannot set an arrow operator as `toolkit_experimental`, this might be causing the problem.